components/Header/Menu/RoomMenu.vue

```diff
@@ -36,7 +36,12 @@
 
     export default {
         computed: {
-            ...mapGetters(['room'])
+            ...mapGetters(['room', 'user']),
+            isRoomOwner() {
+                if (!this.user || !this.room) return false
+
+                return (this.user.id === (typeof this.room.owner === 'string' ? this.room.owner : this.room.owner.id))
```

Comment:
Things that have been working like this. Since in `@cryb/api` it is stored as a string with the user ID and it is passed first as a string, then it gets changed to an Object containing all the user data, it should ensure it doesn't breaks. I don't like how this is being done, though, and I agree it needs a change, but needs a bit of changing how it works.